app/services/create_induction_tutor.rb

```diff
@@ -34,4 +34,19 @@ def start_url
       **UTMService.email(:new_induction_tutor),
     )
   end
+
+private
+
+  def remove_existing_induction_coordinator
+    existing_induction_coordinator = school.induction_coordinators.first
+    return if existing_induction_coordinator.nil?
+
+    if existing_induction_coordinator.induction_coordinator_profile.schools.count > 1
+      existing_induction_coordinator.induction_coordinator_profile.schools.delete(school)
```

Comment:
Just double checking, this deletes the relation between IC and school, right? And cannot possibly delete the school itself?

Reply:
I'll add it to the test 🙂